package.json

```diff
@@ -4737,40 +4737,40 @@
         "language": "csharp",
         "scopes": {
           "namespace": [
-            "entity.name.type.namespace.cs"
+            "entity.name.namespace"
```

Comment:
So looking at the 1.x version, it doesn't specify any fallback textmate scopes for semantic tokens.  Which I believe means that it uses the default fallbacks as defined here - https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#predefined-textmate-scope-mappings .

I think removing the fallbacks for the core LSP semantic token names (aka anything in that table) would have the same affect as what you've done here.  Would you be able to try that?  If it works I think that is the direction we should take.

Reply:
Makes sense, thank you, I've made the change and it looks good!

It's also resolved a 1.x issue I had noticed where local variables had the .cs namespace so looked incorrect in some themes.